target/linux/ramips/image/mt7621.mk

```diff
@@ -490,6 +490,14 @@ define Device/mikrotik_rb750gr3
 endef
 TARGET_DEVICES += mikrotik_rb750gr3
 
+define Device/mikrotik_rb760igs
+  $(Device/MikroTik)
+  DEVICE_MODEL := RouterBOARD RB760i
+  DEVICE_VARIANT := GS
```

Comment:
I see that this is used similarly for rb750, but to me it looks like the GS is part of the device name:
 DEVICE_MODEL := RouterBOARD RB760iGS

Despite, I ask myself whether we should use the long definition names as introduced in ath79 recently here as well:
https://github.com/openwrt/openwrt/commit/6aaa5ce2c5138877e0f0504c3bd536b40e9af928#diff-c231eaae15ac6c91e5a34e4ea305bffaR707

So, Device/mikrotik_routerboard-rb760igs, and thus matching the model name we use anyway.

Reply:
Updated and removed DEVICE_VARIANT.  Changed DEVICE_MODEL to reflect your comment.